docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ax.set_xlabel("Operations")
</pre><p>You can pass this in the bar plot, like: <code>ax = perf_df.plot.bar(color='#7400ff', ylim=(1,200), rot=0, xlabel="Operations", ylabel="Speedup")</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Done